core/services/relay/evm/functions/logpoller_wrapper.go

```diff
@@ -237,17 +237,17 @@ func (l *logPollerWrapper) getCurrentCoordinators(ctx context.Context) (common.A
 	activeCoordinator, err := l.routerContract.GetContractById(&bind.CallOpts{
 		Pending: false,
 		Context: ctx,
-	}, donId, false)
+	}, donId)
 	if err != nil {
 		return common.Address{}, common.Address{}, err
 	}
 
-	proposedCoordinator, err := l.routerContract.GetContractById(&bind.CallOpts{
+	proposedCoordinator, err := l.routerContract.GetProposedContractById(&bind.CallOpts{
 		Pending: false,
 		Context: ctx,
-	}, donId, true)
+	}, donId)
 	if err != nil {
-		return common.Address{}, common.Address{}, err
+		return activeCoordinator, activeCoordinator, nil
```

Comment:
This feels a little weird, but _it is_ the behavior we had before

Reply:
Doesn't sound right to me. If we error out then we send back an error and we won't update the versions at all.

Reply:
If we error out here then the initial `getCurrentCoordinators` halts and we never setup any coordinators, because the initial state when the node starts is:
- routerContract.GetContractById = activeCoordinator
- routerContract.GetProposedContractById = revert

Reply:
I see. Then:
- either change the revert and return zero (I saw that Rens also mentioned it)
- or try to distinguish between a revert due to zero and a different failure
- or return `l.proposedCoordinator `here (i.e. the cached one, which will be initially zero and at some point can be equal to active).